checkstyle/suppressions.xml

```diff
@@ -25,7 +25,7 @@
     <!-- Suppress strict duplicate code checking -->
     <suppress checks="StrictDuplicateCode" files="\.java" lines="1-15"/>
 
-    <suppress checks="IllegalImport" files="com[\\/]hazelcast[\\/]internal[\\/]tpc/"/>
+    <suppress checks="IllegalImport" files="com[\\/]hazelcast[\\/]internal[\\/]tpc"/>
```

Comment:
```suggestion
    <suppress checks="IllegalImport" files="com[\\/]hazelcast[\\/]internal[\\/]tpc[\\/]"/>
```
? like below on line 65?

Reply:
Yeah sure but does it make any difference?